source/libs/get-default-branch.ts

```diff
@@ -33,9 +33,14 @@ async function fetchFromApi(user: string, repo: string): Promise<any> {
 	}
 }
 
-export default function (): Promise<any> {
+export default async function (): Promise<any> {
```

Comment:
I know it's unrelated, but this should probably be
```suggestion
export default async function (): Promise<string> {
```